server/routes/journeys/update-referral/additional-information/schemas.ts

```diff
@@ -0,0 +1,26 @@
+import z from 'zod'
+import { Request, Response } from 'express'
+import { createSchema } from '../../../../middleware/validationMiddleware'
+import { getMaxCharsAndThresholdForAppend } from '../../../../utils/appendFieldUtils'
+
+const TOO_LONG_MSG = (length: number) => `Additional information must be ${length.toLocaleString()} characters or less`
+
+export const schemaFactory = async (req: Request, res: Response) => {
+  const maxLengthChars = req.journeyData.isUpdate
+    ? getMaxCharsAndThresholdForAppend(
+        res.locals.user.displayName,
+        req.journeyData.csipRecord?.referral.otherInformation,
+      ).maxLengthChars
+    : 4000
+
+  const ADDITIONAL_INFORMATION_MSG = 'Enter an update to the additional information'
+
```

Comment:
this variable can be declared outside of the function call

Reply:
Done